app/models/category.rb

```diff
@@ -1,3 +1,9 @@
 class Category < ActiveRecord::Base
-	has_many :videos
+	#has_many :videos, -> { order ("title")}
```

Comment:
Make sure you cleanup commented code. It clutters your real code and is usually not needed.